djstripe/migrations/0003_auto_20181117_2328_squashed_0004_auto_20190227_2114.py

```diff
@@ -0,0 +1,1305 @@
+# Generated by Django 2.1.4 on 2019-02-27 21:37
+
+import django.core.validators
+import django.db.models.deletion
+from django.db import migrations, models
+
+import djstripe.enums
+import djstripe.fields
+
+
+class Migration(migrations.Migration):
+
+	def __init__(self, *args, **kwargs):
+		try:
+			# Hack to support sqlite for quick testing. Without this migrations fail with:
+			#    "django.db.utils.NotSupportedError: Renaming the 'djstripe_paymentmethod' table while in a transaction
+			#    is not supported on SQLite because it would break referential integrity.
+			#    Try adding `atomic = False` to the Migration class."
+			from django.db import connection
+
+			if connection.vendor == "sqlite":
+				self.atomic = False
+		except Exception:
```

Comment:
What's that supposed to catch?

Reply:
It was just to be on the safe side I think I think `connection.vendor` isn't a documented attribute.

Reply:
Use getattr() with a fallback then. A try/catch swallows exceptions.